upstream update

bartimaeusnek opened the pull request.

basdxz reviewed (approved):
Keep the two branches synced, all these changes are good.

POPlol333 committed:
Adding SC UIV and UMV dust recipe + editing a bit of radox line cuz it's completely bullshit (#366)

* fixing random high endgame machine recipe

* add radon polymer fluid pipe

* fixes id ffs this one free, and adjusting a bit non sense radox line to be more reasonable

* craft sc uiv

* fixes recipe dividing amount per 2, didnt ask

* real uiv sc craft

* umv dust

* missclicked helium L whoops

* helium gas changed to liquid version

* take this scala :middlefinger:

GDCloudstrike committed:
Superconductor dtpf recipes (#368)

* Tesseract recipe adjustment

* sc base UIV recipe

* forgot trans metal

* sc base umv dtpf recipe

* Number tweaks

POPlol333 committed:
add craft to infinity egg from witchery (#367)

* add craft to infinity egg from witchery

* now this is MA-GIC

Runakai1 committed:
Moved industrial Apiary to Coremod (#370)

* Moved industrial Apiary to Coremod

* Used Itemlist

chochem committed:
Replace soldering alloy with indalloy for circuits assembler and CAL (#369)

* indalloy for crystal/wetware/bio

* define indalloy earlier

* indalloy for socs and HE

Connor-Colenso committed:
Blacklist (#373)

* Dump dep

* Add blacklist

Co-authored-by: GTNH-Colen <54497873+GTNH-Colen@users.noreply.github.com>

Connor-Colenso committed:
No reversing (#374)

Co-authored-by: GTNH-Colen <54497873+GTNH-Colen@users.noreply.github.com>

Runakai1 committed:
Moves AE2FC recipes to coremod and made them more greggy (#371)

* Tried stuff

* Removed imports

added recursive recipes

* getModItem & OC Condition

* removed always true conditions

* Dependencies gone, ModItem, Rework

* Fixed Build error

* Stuff changed

1k fluid was changed to match 1k item component
Shapeless Recipes for small interface
Added dependency (sorry boubs)

* removed OC condition

* be gone spaces

Dream-Master committed:
Update dependencies.gradle

Dream-Master committed:
Update dependencies.gradle

Dream-Master committed:
Update dependencies.gradle

Glease committed:
gt++ should probably be a compileOnly thing (#376)

* gt++ should probably be a compileOnly thing

* maybe fix gt++ dependency

dipo33 committed:
Add assembler recipes for Advanced Storage Housing and Fluid Storage Housing (#375)

* Add assembler recipes for Advanced Storage Housing and Fluid Storage Housing

* Revert "Add assembler recipes for Advanced Storage Housing and Fluid Storage Housing"

This reverts commit c58e89343da1511d6372bbc2540b6881436d96bb.

* Fix wrongly named variable

* Add assembler recipes for Advanced Storage Housing and Fluid Storage Housing

* Add programmed circuit into the recipes

* Rename the script to better reflect its purpose

* Add comments at the start of AE2FC and EC2 recipes

* Revert "Rename the script to better reflect its purpose"

This reverts commit 4cf2be4e3cd175be092fd3cb89bd7f5a757ece86.

* Revert "Add comments at the start of AE2FC and EC2 recipes"

This reverts commit 574d666b8db124d228df60c47e2a0d7aae532af4.

* Move the recipes into a separate script

S4mpsa committed:
Add recipes for ME Quantum Storage and ME Digital Singularity (#377)

Connor-Colenso committed:
Add new inf recipe (#378)

Co-authored-by: GTNH-Colen <54497873+GTNH-Colen@users.noreply.github.com>

Dream-Master committed:
Update dependencies.gradle

BlueWeabo committed:
Recipesfixed (#379)

greesyB committed:
Replace 4x and 16x co-processor recipes (#380)

Runakai1 committed:
Update GalliumArsenide (#381)

Adde 3 Mixe recipes to turn back to crystal
Changed quantity. 1 Crystal = 2 Dust and vice versa

Dream-Master committed:
bump version

leagris committed:
impr(login_text): refactor more maintainable logint text (#382)

Dream-Master committed:
new welcome text

S4mpsa committed:
Add recipes for ME Input Bus and ME Void Storage (#383)

* Add recipe for ME Input Bus

* Add recipe to ME Void Storage

Dream-Master committed:
Add Oil Rig recipes for IV (#384)

* update BS+Spotless

* update dep file

* add new oilrigg recipes

* fix nei dep

* spotlessApply (#385)

Co-authored-by: Martin Robertz <dream-master@gmx.net>
Co-authored-by: GitHub GTNH Actions <>

* fix recipe derp

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

miozune committed:
Purge OpV (#386)

miozune committed:
Fix crafting recipe for UMV Hull (#387)

miozune committed:
Make UMV Hull not reversible (#388)

Dream-Master committed:
Flower coin recipes (#390)

* add assembler recipes for Flower coins

(cherry picked from commit c22b7c8e2e61da32f53d309ab67e57793abddcc0)

* add more coin exchange recipes

* fix derp

S4mpsa committed:
Fix import to also work on serverside (#391)

Dream-Master committed:
Jabba upgrades (#389)

* add non recursive recipes for Jaba upgrades  (lv-iv)
add control circuit to recipes

* fix recipe derp
add recipes changes to crasfting recipes as well

* fix recipe
change upgrades using a upgrade instead of a barrel

* fix recipe amount

* change upgrade amount

* Use itemlist and ore direction for recipes
nerf upgrades a bit

* make carfting of upgrades longer

* Add recipes for the three new upgrade tiers

* fix recipe derp

* change circuits to UV

Dream-Master committed:
found 4 marshmallow in a Litch castle. That is to much. Lower the rate

Dream-Master committed:
reduce loot a bit

Dream-Master committed:
spotless

Runakai1 committed:
Added Comb Catalyst Recipes (#395)

Quarri6343 committed:
Rework Wooden Form (#393)

* Rework Wooden Form

* spotlessApply (#394)

Co-authored-by: くぁーりぃ <39122497+Quarri6343@users.noreply.github.com>
Co-authored-by: GitHub GTNH Actions <>

* add durability bar and patch dupe

* remove nbt

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

Dream-Master committed:
add two slots to wiremil (#403)

Dream-Master committed:
add a Diode recipe using Wafer and Tantalum wire (#399)

* add a Diode recipe using Wafer and Tantalum wire

* spotlessApply (#400)

Co-authored-by: Martin Robertz <dream-master@gmx.net>
Co-authored-by: GitHub GTNH Actions <>

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

Dream-Master committed:
add recipes from script to core mod (#396)

Dream-Master committed:
removed unused item recipe from Netherstarfragment (#401)

Dream-Master committed:
Put bio ball recipes to code (#397)

Dream-Master committed:
change Si to Sisg (#402)

* change Si to Sisg

* fix more Silicon recipes

(cherry picked from commit 5cde9fa3e9735ad5a4cbcf72453ab67e689e6f35)

* remove duplication recipe

* remove dupe recipe

* add potassium Hydroxide recipe from script
add Rocksalt recipe

* Revert "add potassium Hydroxide recipe from script"

This reverts commit f7eba0d19b8b85e3e0cdde32ca3ea722432c81d3.

* code from another pr

* not belongs in this pr

* spotless

Dream-Master committed:
Sodium potassium (#398)

* add new sodium potassium recipe

* add NAK cell recipes to code

* Add Nak and Helium cells back
change Helium to Liquid Helium (EV available)

* fix Nak and Helium Cells

(cherry picked from commit ae50684e498685d55ca7cd7dfdcc5aebc8a63906)

* add sodium dust recipe

* Revert "add sodium dust recipe"

This reverts commit 1ef1d4e3925b2c44fb1867f2f070121faab052ff.

* remove recipe

* Revert "remove recipe"

This reverts commit 05005a33d6435663dc0b0e0bbe96deddf5488abe.

* add a way back make sodium out of fluid sodium

* add potassium Hydroxide recipe from script
add Rocksalt recipe

(cherry picked from commit f7eba0d19b8b85e3e0cdde32ca3ea722432c81d3)

* spotless

greesyB committed:
Add recipe for NC2 panel memory card (#405)

* Add recipe for NC2 panel memory card

* Spotless apply

boubou19 committed:
add back the /dev/null (#406)

boubou19 committed:
fix recipe conflict with void drop filter

Dream-Master committed:
Nano armor (#407)

* Add control circuits to Nano Armor
change molten Redstone to molten Redstone Alloy

* Add Energestic Alloy Wire to Nano Armor

* add energetic fine wire to Nano Armor
add a fe Core miod items to itemlist
change Carbon parts now made in the forming press with Armor molds

* fix derp

LewisSaber committed:
Fix Glass Fiber Wiremill Recipe (#408)

* Fix Recipe to match extruder an other fine wires

* Add Circuit #3

* Update dependencies

* Exchange circuit and material slot

Shadowsoftime99 committed:
Hid fake circuits from NEI (#409)

* Hid the fake circuits from NEI

* Spotless fixes

Alexdoru committed:
minor text change (#410)

GlodBlock committed:
rad hatch compact (#411)

Quarri6343 committed:
Add Compressed Chest assembler recipe (#412)

* add compressed chest assembler recipe

* fix

Dream-Master committed:
Change Wooden Brickform durability (#413)

* Change Wooden Brickform durability

Change Wooden Brickform durability to 10 stacks (640)

* Update WoodenBrickForm.java

Runakai1 committed:
Dtpf bee part2 (#414)

* Changed plasma discount

* Apply new Discount to CSNT too

* remove added dependencies

GDCloudstrike committed:
Change OCD pixels (#415)

llk89 committed:
convert ticon dusts to gt dusts in some recipes (#416)

Co-authored-by: Glease <4586901+Glease@users.noreply.github.com>

Dream-Master committed:
Update ScriptAvaritiaAddons.java

And i wonder why i got an obsidian chest :>(

Dream-Master committed:
Update ScriptAvaritiaAddons.java

Upgrade Obsidian to Compressed chest

Dream-Master committed:
fix recipes and add new one

Dream-Master committed:
update BS+SA

Dream-Master committed:
fix recipe

Dream-Master committed:
fix circuit

boubou19 committed:
added extreme shaped recipe for the creative staff of travelling (#417)

Dream-Master committed:
Clean room (#418)

* add assembler recipes for cleanroom components

* move recipe to mv

miozune committed:
Fix recursive recipes for ae2fc storage components being cheaper than assembler recipes (#419)

* Fix recursive recipes for ae2fc storage components being cheaper than assembler recipes

* spotlessApply (#420)

Co-authored-by: miozune <miozune@gmail.com>
Co-authored-by: GitHub GTNH Actions <>

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

ChromicRedBrick committed:
Delete duplicate key (#421)

Co-Authored-By: Kiwi <42833050+Kiwi233@users.noreply.github.com>

Co-authored-by: Kiwi <42833050+Kiwi233@users.noreply.github.com>

Connor-Colenso committed:
Buff some steam age recipes (mostly time buffs) (#422)

* Buff flint maceration time.

* Buff some early alloy smelting recipes

Co-authored-by: GTNH-Colen <54497873+GTNH-Colen@users.noreply.github.com>

miozune committed:
Use the same number for programmed circuit and item meta except for 0 in Ztones recipes (#423)

Dream-Master committed:
bump version

GlodBlock committed:
inductor usage (#424)

Dream-Master committed:
Indusctors (#425)

* add Inductor recipes

* spotlessApply (#426)

Co-authored-by: Martin Robertz <dream-master@gmx.net>
Co-authored-by: GitHub GTNH Actions <>

* change induscor smd recipes

* change to conductive iron

* change xsmd materials for inductors

* update BS+SA

* Revert "update BS+SA"

This reverts commit 04a5e3b004896d88b58a68e96d95bae91857e7bc.

* update BS

* SA

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

Dream-Master committed:
fix derp

(cherry picked from commit 1f9f2827ff739f2135251e2c11cd6099c9065d65)

kuba6000 committed:
Move upgrade recipes from GT5 and scripts (#429)

* Move recipes

* Update dependencies.gradle

* Update dependencies.gradle

Co-authored-by: Martin Robertz <dream-master@gmx.net>

Dream-Master committed:
bump version

Dream-Master committed:
Merge branch 'master' of github.com:GTNewHorizons/NewHorizonsCoreMod

S4mpsa committed:
Adds recipes for UIV and UMV singleblock machines (#430)

* put in recipes to the machines

* fix UMV cable issue

* Change a few materials around

* Update dependencies.gradle

Co-authored-by: BlueWeabo <76872108+BlueWeabo@users.noreply.github.com>
Co-authored-by: Martin Robertz <dream-master@gmx.net>

kuba6000 committed:
Catch Thaumcraft Fake Player (#432)

miozune committed:
Remove circuit 0 (#431)

Dream-Master committed:
bump version

miozune committed:
Remove Zincite EBF recipe (#433)

Runakai1 committed:
Replace Poison Ivy with cacti in upgrade (#434)

GlodBlock committed:
recipe for me output hatch and more superconductor coils (#435)

Dream-Master committed:
BS+SA

Dream-Master committed:
Update dependencies.gradle

POPlol333 committed:
Tps card craft (#436)

* add recipe for tps card in circuit assembler

* remove unused import

POPlol333 committed:
spotless (#437)

Co-authored-by: boubou19 <miisterunknown@gmail.com>

POPlol333 committed:
hypogen->infinity coil (#438)

Steelux8 committed:
Changed Coil Recipes to the New Material (#363)

* Changed Coil Recipes to the New Material

- Changed input materials to match the other PR about the EV coil change.

* Add ebf and vac freezer recipe for TPV

(cherry picked from commit 01f2e915722683bbe0bb5ae2dc00d987eabddcf8)

Co-authored-by: Martin Robertz <dream-master@gmx.net>

Dream-Master committed:
Update dependencies.gradle

Dream-Master committed:
AFB should use all iv circuits (#441)

BS+SA

Dream-Master committed:
Tantalum change (#440)

* Buff Tantalum

* remove ender io capacitor recipes
add them back via assembler

* remove recursive capacitor recipes

* revert

* change mod name variable

* change silver capacitor recipe

* update BS+SA

* remove last recursive recipe

* change Gravistar->Quantumstar

minecraft7771 committed:
Added recipe generation for cable diodes UHV and above (#442)

Dream-Master committed:
Update build.gradle

Dream-Master committed:
remove pure alu to sodium alunite recipes (#443)

* remove pure alu to sodium alunite recipes
change sodium alunite recipe gives always 64 items per craft
change red mud amount per craft
change red mud recipe from 10k to 1K
add the sodium alumitae mixer recipes to  chem reactor recipes (pure variants)

* spotlessApply (#444)

Co-authored-by: Martin Robertz <dream-master@gmx.net>
Co-authored-by: GitHub GTNH Actions <>

* change Co2 to 4k

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

OneEyeMaker committed:
Greggified recipes of fluid storage cells (#445)

BlueWeabo committed:
add transformers (#446)

miozune committed:
Rewrite GUIs with ModularUI (#447)

miozune committed:
Make hazmat-proof armors protect player from custom hazard effects (#448)

chochem committed:
Fix Honey for Biomass (#449)

* honeyfix

* spotless

BlueWeabo committed:
Add 2 new lenses for the Nano Forge (#450)

* add 2 new lenses and their recipes

* found the issue

* add radox polymer to the PCB material handler

Co-authored-by: Martin Robertz <dream-master@gmx.net>

Glease committed:
add reinforced glass alfglass managlass as boroglass replacement in multi structure (#451)

Alrightsc committed:
Add assembler recipes for Impregnated Frame, Apiary, Scented Paneling, all Alveary upgrades (#452)

* Added bunch of carpenter recipes as assembler recipes

* spotless

Dream-Master committed:
update version

BritishCynic committed:
Change JABBA circuits to 13 (#454)

This changes all JABBA upgrade recipes in the Assembler to 13 to avoid conflicts.

chochem committed:
Make MV cooling lossless on coolant (#455)

* buff MV cooling

* revert to only changing coolants. lossless now

* one number was forgotten

BritishCynic committed:
Add remaining IronChest Assembler options (#456)

* Add remaining IronChest Assembler options

Provides more comprehensive options for https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11226.

Adds new options for constructing Steel, Iron, Gold and Diamond Chests by allowing previous chests to be used with their existing upgrade materials from the manual upgrades.

* SpotlessApply

SpotlessApply

* Second SpotlessApply

Second time with nothing else committed afterward.

* Update build script

Update build script

Co-authored-by: Martin Robertz <dream-master@gmx.net>

Runakai1 committed:
Adds Level Maintainer Recipe to AE2FC (#457)

miozune committed:
Adapt Nether Star wire to Wiremill Configuration Circuit support (#458)

chochem committed:
fix nether star (#459)

Dream-Master committed:
update version and uupdate Text

Dream-Master committed:
Update LoginHandler.java

asdflj committed:
add fluid terminal recipe (#460)

add fluid buffer recipe

Dream-Master committed:
update text

Dream-Master committed:
Merge branch 'master' of github.com:GTNewHorizons/NewHorizonsCoreMod

# Conflicts:
#	src/main/java/com/dreammaster/loginhandler/LoginHandler.java

Dream-Master committed:
Add a cheaper recipe with Helium Gas for SiSG ingots in EBF (#465)

(cherry picked from commit 14f0dcb6f758d90dd5377cf57e3bb3ceb87b522d)

Dream-Master committed:
Mining pipe recycling (#461)

* Mining pipe recycling

* spotlessApply (#462)

Co-authored-by: Martin Robertz <dream-master@gmx.net>
Co-authored-by: GitHub GTNH Actions <>

* fix mining pipe

* fix arc furnace recipe again

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

Dream-Master committed:
add smd inductor recipe variant (#463)

* add smd inductor recipe variant

* spotlessApply (#464)

Co-authored-by: Martin Robertz <dream-master@gmx.net>
Co-authored-by: GitHub GTNH Actions <>

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
Co-authored-by: Connor-Colenso <52056774+Connor-Colenso@users.noreply.github.com>

POPlol333 committed:
add ducttape recipe (#466)

* add ducttape recipe

* spotlessApply (#467)

Co-authored-by: RIONDY 'POPlol333' Adam <76914762+POPlol333@users.noreply.github.com>
Co-authored-by: GitHub GTNH Actions <>

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
Co-authored-by: Connor-Colenso <52056774+Connor-Colenso@users.noreply.github.com>

BlueWeabo committed:
add the shapes (#468)

BlueWeabo committed:
fix (#469)

Pxx500 committed:
Fix too many zeres (#470)

S4mpsa committed:
Add recipe for Hyper-Acceleration Card (#471)

* Add recipe for Hyper-Acceleration Card

* spotlessApply (#472)

Co-authored-by: Sampsa <69092953+S4mpsa@users.noreply.github.com>
Co-authored-by: GitHub GTNH Actions <>

Co-authored-by: Martin Robertz <dream-master@gmx.net>
Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

POPlol333 committed:
change 300s to 180s (#473)

GDCloudstrike committed:
Add totemic capacitor recipes (#474)

Co-authored-by: Martin Robertz <dream-master@gmx.net>

Runakai1 committed:
Add Roof Tile (TF) recipe (#476)

+ rearranged order of script loading to keep consistency

POPlol333 committed:
Sc uiv rebalance (#475)

* update gt5 dep

* radox polymer pipe to trans metal

* fix radox polymer lcr recipe

* fixed quantity

Co-authored-by: Martin Robertz <dream-master@gmx.net>

POPlol333 committed:
Add UXV Energy Hatch (without recipe) (#477)

* add the hatch to the existence realm

* spotlessApply (#478)

Co-authored-by: RIONDY 'POPlol333' Adam <76914762+POPlol333@users.noreply.github.com>
Co-authored-by: GitHub GTNH Actions <>

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

Dream-Master committed:
Add Placeholder recipes for UXV Hulls and Caings (#479)

* Add Placeholder recipes for UXV Hulls and Caings

* spotlessApply (#480)

Co-authored-by: Martin Robertz <dream-master@gmx.net>
Co-authored-by: GitHub GTNH Actions <>

* forgot recipe

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

RaitGamerGR committed:
Hi-Amp Transformers recipe unification (#481)

* Transformers

Unified wires on the transformers recipes.

* Changed recipe fix

Dream-Master committed:
bump version

GDCloudstrike committed:
Add recipes to the nei planet blocks (#483)

* Add totemic capacitor recipes

* Add planet recipes

* Add programmed circuits

Dream-Master committed:
no 0 config

Dream-Master committed:
update BS

chochem committed:
Fix compressed chest conflict (#484)

* fix compressed chest conflict

* reduced obsidian

Phineasor committed:
Creative essentia cell (#486)

* Add Thaumic Energistics class and a recipe for Creative essentia cell.

* Update ScriptThaumicEnergistics.java

Made the essentia the correct numbers.

* SpotlessApply.

* ScriptLoader updated to load the script.

* Mod dev instance runs with all needed mods for the recipe to work. Also adds research for cell.

* Remove added test deps, fixed NPE crash, and finished the recipe.

Co-authored-by: Martin Robertz <dream-master@gmx.net>

GDCloudstrike committed:
Chromatic glass dtpf recipe (#485)

Co-authored-by: Martin Robertz <dream-master@gmx.net>

GDCloudstrike committed:
Fix math (#487)

GlodBlock committed:
recipe for ae2fc's fluid storage bus (#491)

* ae2fc storage bus recipe

* sa

Dream-Master committed:
Quad Input Hatch recipes (#488) (#489)

* Quad Input Hatch recipes (#488)

* bump gt version

* Optical Boule

* Wafers

* Optical Processor

* Revert "Optical Processor"

This reverts commit 163a59b849d2f89dbf2bc5e23349405a284e1b0c.

* Revert "Wafers"

This reverts commit bafb3d1888b8032ef7e0f4d41e0e4055c7e11b43.

* Revert "Optical Boule"

This reverts commit dcd037a32cfb3a3a902e37e82108df6e94f9f0c4.

Co-authored-by: TheEpicGamer274 <102255081+TheEpicGamer274@users.noreply.github.com>
Co-authored-by: GDCloud <gdcloudstrike@gmail.com>
Co-authored-by: GTNH-Colen <54497873+GTNH-Colen@users.noreply.github.com>

D-Cysteine committed:
Replace creative staff of travelling with teleport staff (#494)

* Replace creative staff of travelling with teleport staff

See:
 * https://github.com/GTNewHorizons/EnderIO/pull/89

* Update dependencies.gradle

---------

Co-authored-by: Martin Robertz <dream-master@gmx.net>

GlodBlock committed:
add covert recipe between ec and ae large cell (#493)

* add covert recipe between ec and ae large cell

* sa+bs

---------

Co-authored-by: Martin Robertz <dream-master@gmx.net>

Dream-Master committed:
bs

chochem committed:
apiary with oredict (#497)

GDCloudstrike committed:
Optical Circuit Recipes (#490)

* Quad Input Hatch recipes (#488)

* bump gt version

* Optical Boule

* Optical wafers

* Optical processor

* Spotless

* merge master  (#492)

* recipe for ae2fc's fluid storage bus (#491)

* ae2fc storage bus recipe

* sa

* Quad Input Hatch recipes (#488) (#489)

* Quad Input Hatch recipes (#488)

* bump gt version

* Optical Boule

* Wafers

* Optical Processor

* Revert "Optical Processor"

This reverts commit 163a59b849d2f89dbf2bc5e23349405a284e1b0c.

* Revert "Wafers"

This reverts commit bafb3d1888b8032ef7e0f4d41e0e4055c7e11b43.

* Revert "Optical Boule"

This reverts commit dcd037a32cfb3a3a902e37e82108df6e94f9f0c4.

Co-authored-by: TheEpicGamer274 <102255081+TheEpicGamer274@users.noreply.github.com>
Co-authored-by: GDCloud <gdcloudstrike@gmail.com>
Co-authored-by: GTNH-Colen <54497873+GTNH-Colen@users.noreply.github.com>

Co-authored-by: Yang Xizhi <60341015+GlodBlock@users.noreply.github.com>
Co-authored-by: TheEpicGamer274 <102255081+TheEpicGamer274@users.noreply.github.com>
Co-authored-by: GDCloud <gdcloudstrike@gmail.com>
Co-authored-by: GTNH-Colen <54497873+GTNH-Colen@users.noreply.github.com>

* Adjust recipe time

* moved optical processor to tectech

* Recipe adjustments + Comments

* Optically Perfected CPU

* CPU Containment Housing

* bump gt dep

* Spotless

* add Radon fallback

* Spotless

* Expand LuV-UMV forming press/PLE slots

* Nano to Optical mainframe conversion

* Change nano to optical

* Nano -> any UIV

* no softlocks

* adjust time

* Water line recipes

* Spotless Apply

* Forge hammer chip recipe

* Fixes + Solar Splitter & ASP Irradiant Panel Assembler recipes.

* small cleanup

* dumb spotless

* Grade 8, Glowstone & Cleanup

* Recipe Changes

* Fixes

* Fix 0 items

* Spotless

* Water adjustments

* no more copy amount

---------

Co-authored-by: TheEpicGamer274 <102255081+TheEpicGamer274@users.noreply.github.com>
Co-authored-by: Martin Robertz <dream-master@gmx.net>
Co-authored-by: Yang Xizhi <60341015+GlodBlock@users.noreply.github.com>
Co-authored-by: GTNH-Colen <54497873+GTNH-Colen@users.noreply.github.com>
Co-authored-by: BlueWeabo <76872108+BlueWeabo@users.noreply.github.com>

GDCloudstrike committed:
Fix sifter crash + add chance outputs (#500)

* bump dep

* fix water + add chance outputs

Ethryan committed:
Add Reinforcement fluid Solidifier Recipe (#495)

* Add Reinforcement fluid Solidifier Recipe

* Change Fluid

* 2 ingots

---------

Co-authored-by: Martin Robertz <dream-master@gmx.net>

mitchej123 committed:
[ci skip] Update buildscript to RetroFuturaGradle

mitchej123 committed:
[ci skip] spotlessApply with the new settings

mitchej123 committed:
[ci skip] Add git-blame-ignore-revs for spotlessApply

Dream-Master committed:
update dep

Dream-Master committed:
bs

boubou19 committed:
Sanitize recipes (#501)

* moved dtpf recipes

* moved bending machine recipes

* moved fluid solidifier recipes

* moved distillery recipes

* moved implosion compressor recipes

* moved mixer recipes

* moved some assembler and circuit assembler recipes

* moved chembath recipes

* moved chemical reactor recipes

* renamed classes

* moved compressor class

* moved fluid canner recipes

* alloy smelter recipes

* canner recipes

* centrifuge recipes

* assline recipes

* recipe removal

* autoclave recipes

* blastfurnace recipes

* brewingmachine recipes

* electrolyzer recipes

* laser engraver recipes

* distillation tower recipes

* forming press recipes

* lathe recipes

* disassembler blacklist

* pulverizer recipes

* space autoclave recipes

* wiremill recipes

* forgeHammer recipes

* cutting machine recipes

* sifting recipes

* fluid extractor recipes

* extractor and extruder recipes

* fermenter recipes

* vacuum freezer recipes

* another compressor recipe

* another wiremill recipe

* spotless

* arc furnace recipes

* finish initial splitting

GDCloudstrike committed:
missing forge hammer recipe (#502)

chochem committed:
Fix around 100 critical recipes (#503)

* Fix 100 critical recipes

* sa

Ethryan committed:
Lock behind steel instead (#508)

GlodBlock committed:
move ec recipes to ae/ae2fc (#506)

* move ec recipes to ae/ae2fc

* spotlessApply (#507)

Co-authored-by: GitHub GTNH Actions <>

---------

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
Co-authored-by: Martin Robertz <dream-master@gmx.net>

BritishCynic committed:
Update NameRemover.java (#504)

* Update NameRemover.java

This adds new functionality to the Name Remover

* Can now remove dyes from armor
* Can now remove the RepairCost NBT tag
* Can now remove the "color" NBT tag used when Railcraft tank blocks are removed from the world

* SpotlessApply

* Update NameRemover.java

Update logic, add ability to remove spray colour from GT items and separated out ability to remove dye into its own check.

SA also run

---------

Co-authored-by: Martin Robertz <dream-master@gmx.net>

TheEpicGamer274 committed:
Radox changes (#505)

* Radox buffs in progress

* Radox buffs in progress

* Spotless Apply

* Removing/Fixing Old Recipes in BacteriaRegistry and adding a new CrackingRecipes.java for all future cracking recipes.

---------

Co-authored-by: Martin Robertz <dream-master@gmx.net>

Dream-Master committed:
update bs
remove ae2 warning. We want bug report.

ReignOfFROZE committed:
Added carbon monoxide output to rutile EBF recipe (#509)

Tyrantnh committed:
Add recipe for wetware SoC and wetware with SoC (#510)

* Update CircuitAssemblerRecipes.java

* Update LaserEngraverRecipes.java

Dream-Master committed:
udate bs + sa

Dream-Master committed:
Update dependencies.gradle

Dream-Master committed:
fix error

Dream-Master committed:
fix

chochem committed:
forgehammer fix (#512)

asdflj committed:
add wireless interface terminal recipe (#513)

* add fluid terminal recipe
add fluid buffer recipe

* add wireless interface terminal recipe

chochem committed:
inf chest fix (#514)

Co-authored-by: Martin Robertz <dream-master@gmx.net>

GDCloudstrike committed:
Add some more outputs to water line (#515)

chochem committed:
fix new component recipes (#517)

Hexasphere0 committed:
remove tiny dusts (#519)

Dream-Master committed:
update warning text

Dream-Master committed:
update bs+sa

miozune committed:
Disable recycling concrete bucket (#520)

Alexdoru committed:
move the ASM to remove bibliocraft recipes from Hodgepodge to Dreamcraft (#523)

Alexdoru committed:
Small rework of the IScriptLoader interface (#521)

* quick fix in IScriptLoader

* unnecessary comma

* remove static fields to get script name and dependencies with interface method

* remove interface methods that gets the time to load the script

* spotlessApply (#522)

Co-authored-by: GitHub GTNH Actions <>

---------

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
Co-authored-by: boubou19 <miisterunknown@gmail.com>

LewisSaber committed:
Move some GC script to coremod + add rocket parts assembler recipes (#518)

* Add Assembler recipes for recoket parts/ add GC crafting table recipes removing to coremod

* Remove newlines

* Spotless Apply

---------

Co-authored-by: Martin Robertz <dream-master@gmx.net>

Dream-Master committed:
bump version

Alexdoru committed:
Setup ASM tools (#525)

* move ASM transformers to own package

* make one class implement the IClassTransformer interface and create another interface for transforming classes

* move the ItemFocusWardingTransformer to the new ASM system

* remove comments

* make a package for transformers whose purpose are to remove recipes

* spotlessApply (#526)

Co-authored-by: GitHub GTNH Actions <>

---------

Co-authored-by: Martin Robertz <dream-master@gmx.net>
Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

TheEpicGamer274 committed:
Fixing recipe output amount (#528)

Ethryan committed:
Update en_US.lang (#529)

LewisSaber committed:
Fix Hull inside quad UIV hatch (#531)

Alexdoru committed:
Move TravellersGear recipes to dreamcraft (#527)

Ethryan committed:
Chang drill head shape recipe (#532)

Co-authored-by: Martin Robertz <dream-master@gmx.net>

GDCloudstrike committed:
Nether star pipe extruding (#533)

GDCloudstrike committed:
Add Space assembler recipe file + recipes for optical components (#530)

* Space assembler recipes

* remove circuit assembler recipe

* update dep

* run SpAss recipes

---------

Co-authored-by: Martin Robertz <dream-master@gmx.net>

miozune committed:
Use new method for tiered tooltips (#534)

Dream-Master committed:
bump version nr

Dream-Master committed:
bs+sa

Dream-Master committed:
correct version

RaitGamerGR committed:
Iridium Neutron Reflector recipes skip (#536)

Added high-tier 1 step recipe for Iridium Neutron Reflector.

Glease committed:
Delete HoverModeFix and move it to GraviSuiteNeo (#535)

* change target class name

* delete HoverModeFix.java since it is now fixed in GraviSuiteNeo

---------

Co-authored-by: Alexdoru <57050655+Alexdoru@users.noreply.github.com>
Co-authored-by: Martin Robertz <dream-master@gmx.net>

Dream-Master committed:
bs+sa

asdflj committed:
add ae2fc cells recipe (#537)

* add fluid terminal recipe
add fluid buffer recipe

* add wireless interface terminal recipe

* add ae2fc cells recipe

* Update ScriptAE2FC.java

* Update ScriptAE2FC.java

spotless

---------

Co-authored-by: Martin Robertz <dream-master@gmx.net>

Steelux8 committed:
Fix for Tungstensteel Ingots Needing TPV Coils (#538)

- Corrected the appropriate recipes in order to set EBF temperature to 4000K, as intended by the last change.

minecraft7771 committed:
Move space assembler calls to GTNH intergalactic (#539)

Dream-Master committed:
bump version

Dream-Master committed:
update warn text

eigenraven committed:
Update buildscript and dependencies (#540)

* Update buildscript and remove unnecessary transformer exclusion

* Clean up dependencies

* Update dependencies

* Update gradle.properties, switch to RFG tag class generation

* Force enable mixins (gt5u uses them transitively)

* Update yamcl to fix runObfClient

* Print count of disabled and found dependencies in the log

Dioxop committed:
Added macerator, fluid extractor recipes for Glass Bottles (#541)

Co-authored-by: Dioxop <Dioxop.protonmail.com>
Co-authored-by: Martin Robertz <dream-master@gmx.net>

Dream-Master committed:
bs+sa

Dream-Master committed:
Ender io recipes (#542)

* update bs+sa

* add Me conduit recipes to code

* remove me conduits crafting recipes

* change Ultra dense me conduit recipe use molten silicone

* change gt fluids to ender io fluids
add Circuit to conduits
change ratio 16->4

* spotlessApply (#543)

Co-authored-by: GitHub GTNH Actions <>

* change ratio to 1:1

* sa

* derp

---------

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

Dream-Master committed:
Ender io recipes (#544)

* update bs+sa

* add Me conduit recipes to code

* remove me conduits crafting recipes

* change Ultra dense me conduit recipe use molten silicone

* change gt fluids to ender io fluids
add Circuit to conduits
change ratio 16->4

* spotlessApply (#543)

Co-authored-by: GitHub GTNH Actions <>

* change ratio to 1:1

* sa

* derp

* add control circuit

* sa

---------

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

Dream-Master committed:
Ae2 cables change (#545)

* Change Ae2 cable recipes

(cherry picked from commit 86f970b537328787eb5bb91f9d45a551cc291a59)

* add ae2 cable recipe to code

* remove dey recipes because we used the ae2 vanilla ones
try to add recipes shorter

* derp

* derp

* add two more recipes so covered cables can become smart one

* add covered cable to smart cable

GDCloudstrike committed:
Add recipe for humongous input hatch (#546)

* update dependencies

* recipe

Dream-Master committed:
bump version
clean imports

Dream-Master committed:
update buildscript

Dream-Master committed:
update dep file

Runakai1 committed:
Adjust according to tier (#547)

* Adjust according to tier

* Spotless

kuba6000 committed:
Add recipe to night upgrade (#549)

* Bump

* Add recipe to night upgrade

chochem committed:
fix (#550)

chochem committed:
fixrocketrecipes (#551)

boubou19 committed:
Cleaning up code (#552)

* remove dead code

* bump GT5U version

* use enums (part 1)

* spotless

* use enums (part 2)

* bump GT version

* migrate from the temmporary enum to the new enum

* migrate forgotten mod IDs to GT5U enum

* make it compile

boubou19 committed:
remove silly scala import (#553)

GDCloudstrike committed:
Add new bio circuit recipe aka bio SoC + adjust wetware SoC (#554)

* bio wafer

* bio chips

* bio circuits

* adjust wetware soc time

* 50

* move to RA2 and address reviews

chochem committed:
fix ae recipes (#555)

chochem committed:
Some more extracells cleanup (#558)

* fix incorrect ids

* remove ec recipes

* remove ec enum import

* Revert "remove ec recipes"

This reverts commit fee15e1c5c085e5c450f767e9ad53cd42fa11e3c.

* change outputs from ec to oc and order correctly

* sa

Connor-Colenso committed:
Kill Disassembler in Core Mod (#556)

* Purge disassembler

* Change arc furnace on dreams request.

* spotlessApply (#557)

Co-authored-by: GitHub GTNH Actions <>

---------

Co-authored-by: GTNH-Colen <54497873+GTNH-Colen@users.noreply.github.com>
Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
Co-authored-by: Martin Robertz <dream-master@gmx.net>

Dream-Master committed:
Update dependencies.gradle

DaCurse committed:
Add Assembler recipes for Chisel blocks (#560)

* Add moved assembler recipes from zenscript

* Added Chisel block recipes to Assembler

* Change balancing a bit

* Fix road line recipe

* Run spotless

chochem committed:
AsGa fluid extraction (#561)

Dream-Master committed:
Update SpaceAssemblerRecipes.java

Dream-Master committed:
Update SpaceAssemblerRecipes.java

Dream-Master committed:
Update dependencies.gradle

Dream-Master committed:
update bs+sa

minecraft7771 committed:
Fix space assembler recipes (#563)

* Added missing import

GDCloudstrike committed:
Some UXV additions (#564)

* Purge disassembler

* Change arc furnace on dreams request.

* spotlessApply (#557)

Co-authored-by: GitHub GTNH Actions <>

* add uxv variants

* recipes

---------

Co-authored-by: GTNH-Colen <54497873+GTNH-Colen@users.noreply.github.com>
Co-authored-by: Connor-Colenso <52056774+Connor-Colenso@users.noreply.github.com>
Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

Pxx500 committed:
Alumina and silicon dioxide changes (#562)

* Recipes transfered from materials,java

* Fix

* Update ElectrolyzerRecipes.java

* Update ElectrolyzerRecipes.java

* Update ElectrolyzerRecipes.java

* Fix recipe time and voltage

* Fix mistake

* Fix

* spotless + oxygen

* Update ElectrolyzerRecipes.java

* mica fix

* Zeolite change

* changes

* oversight

* Pollucite

* vermiculite

* the compound rule applied

---------

Co-authored-by: kuba6000 <kuba.123123.6000@gmail.com>
Co-authored-by: Martin Robertz <dream-master@gmx.net>

Dream-Master committed:
Update dependencies.gradle

Dream-Master committed:
Clay cast (#565)

* add clay cast recipes

* spotless

* fix

kuba6000 committed:
Move all TC researches, recipes from scripts (#482)

* Begin working on TC research

* build test

* Spoootless

* fixz

* Helper method to create ItemStacks with NBT

* Recipe remover

* More recipe removers

* ERROR: CODE TOO LARGE

* Its working???

* Update en_US.lang

* Add null checks

* Null check

* Check for ItemStack

* Forgot one

* Fix blood caps research

* EMT

* Revert "EMT"

This reverts commit 3f21858711d009e0bc5f547950e5bfd2d7c39261.

* Run scripts later for now

* Fix setResearchAspects

* Fix null name in logs

* Add own category in Init

Dream-Master committed:
Fix materials (#566)

* update dep file
update materials

(cherry picked from commit f689cd37bd4637fa6a76773dbfb0c05ff0c30b1c)

* update materials

(cherry picked from commit bd2823f9944271e3473d0e313d47d63e10cb0062)

* bs+sa

(cherry picked from commit fce2a5707199c821d50a61badc5a36decb45e338)

* add magic bees

* add magic bees

* Use stack comparasion

* deps

* deps2

* deps3

* deps4

* ??????

* deps5

* deps6

* deps7

* deps8

* deps9

* deps10

* deps11

* deps12

* deps13

* deps14

* deps15

* deps16

* deps17

---------

Co-authored-by: kuba6000 <kuba.123123.6000@gmail.com>

kuba6000 committed:
Fix crash (#567)

* Ignore NBT while searching for recipes

* test

* Update dependencies.gradle

* Use mods enum

* Better missing itemstack

* Revert "test"

This reverts commit 0511d536122f0383f2ec3aeae5909de5624ba3a9.

* dep

---------

Co-authored-by: chochem <40274384+chochem@users.noreply.github.com>

GDCloudstrike committed:
Add Stellar Catalyst DTPF recipes (#570)

* stellar cat recipes

* move recipes

---------

Co-authored-by: Martin Robertz <dream-master@gmx.net>

Dream-Master committed:
Update dependencies.gradle

Dream-Master committed:
Revert "Update dependencies.gradle"

This reverts commit 325c5c045d939fba0a3f2ef888f308b852560006.

Dream-Master committed:
Update dependencies.gradle

Dream-Master committed:
update pack version

chochem committed:
oredict diodes and inductors (#571)

glowredman committed:
Enable GalacticGreg oregen for Amun-Ra (#568)

* Enable GalacticGreg oregen for Amun-Ra

* Apply spotless

* spotlessApply (#569)

Co-authored-by: GitHub GTNH Actions <>

---------

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
Co-authored-by: Martin Robertz <dream-master@gmx.net>

Dream-Master committed:
fix https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/13314

Dream-Master committed:
fix sieve (#573)

* fix https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/13300

* spotlessApply (#574)

Co-authored-by: GitHub GTNH Actions <>

---------

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

Dream-Master committed:
Fix Phial recipe (#572)

* fix https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/13299

* move recipe

* change recipe

* change recipe again

* add rubber variants

* sa

* Add shapeless

* Use itemstack instead of item

---------

Co-authored-by: kuba6000 <kuba.123123.6000@gmail.com>

Pxx500 committed:
Chemical Rebalance™ (#576)

mitchej123 committed:
ForgeDirection (#577)

* ForgeDirection

* update dep

* update bs

---------

Co-authored-by: Martin Robertz <dream-master@gmx.net>

kuba6000 committed:
Move everything from the scripts to Core Mod (#575)

* Add new methods to remove recipes

* Update GT_Recipe_Remover.java

* Remove recipes

* AE2.zs

* AE2.zs

* Adv-Solar-Panel.zs

* Advanced-Backpacks.zs

* Avaritia.zs

* Backpack.zs

* Better-Builders-Wands.zs

* Better-Questing.zs

* Binnies-Core.zs

* Binnies-Mods.zs

* Biome-o-Plenty.zs

* Blood-Arsenal.zs

* Blood-Magic.zs

* Botany.zs

* Build-Craft.zs

* Carpenters-Blocks.zs

* Cat-Walk.zs

* Chisel.zs

* Compact-Kinetic_generators.zs

* CoreMod.zs

* Draconic-Evolution.zs

* Extra-Cells.zs

* Emt.zs

* Ender-IO.zs

* Enhanced-Loot_Bags.zs

* Extra-Bees.zs

* Extra-Trees.zs Extra-Trees2.zs Extra-Trees3.zs

* Extra-Utilities.zs

* Flood-Light.zs

* Forbidden-Magic.zs

* Forestry.zs

* Update ScriptLoader.java

* Fix error: code too large

* Allow putting less than 9 items in shaped recipe

* GT++.zs

* Gadomancy.zs

* Galacticraft.zs

* Fix method too large error

* Update ScriptCoreMod.java

* Gendustry.zs

* Genetics.zs

* GraviSuite.zs

* Gregtech.zs

* HardcoreEnderExpansion.zs

* Harvestcraft.zs

* Harvestcraft-Nether.zs

* IC2.zs

* Iron-Chests-Minecarts.zs

* Iron-Chests.zs

* Logistics-Pipes.zs

* Magic-Bees.zs

* Malis-Doors.zs

* Mechworks.zs

* Mine-and-Blade-Battlegear2.zs

* Minecraft.zs

* Natura.zs

* Nuclear-Control.zs

* Open-Blocks.zs

* OpenModularTurrets.zs

* Project_Red.zs

* Railcraft.zs

* Random-Things.zs

* RemoteIO.zs

* Steves-Carts.zs

* Steves-Factory-Manager.zs

* SuperSolarPannel.zs

* Thaumcraft-01-Basic.zs Thaumcraft-02-Thaumaturgy-1.zs Thaumcraft-03-Alchemy.zs Thaumcraft-04-Artifice.zs

* Thaumic-Bases-01-Main.zs

* Thaumic-Energistics.zs

* ThaumicTinkerer-01.zs

* ThaumicTinkerer-02-Kami.zs

* Tinkers-Construct.zs

* Tinkers-Defence.zs

* TwilightForest.zs

* Wireless-Redstone.zs

* Witchery.zs

* ZZ-Client-Only.zs

* Fix code too large errors

* Try to load every script before crashing

* Fix recipe errors

* Readd recipes that got accidentally removed in https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/pull/13232

* Separate AE2 Wireless Crafting from AE2

* Add Chisel Helper

* Add more helper methods

* All of the NEI API calls from scripts

* Backpack.zs

* Binnies-Core.zs

* Biome-o-Plenty.zs

* Blood-Arsenal.zs

* Blood-Magic-Thaumcraft.zs

* Blood-Magic.zs

* Botany.zs

* Chisel.zs

* CoreMod.zs

* Extra-Bees.zs

* Extra-Trees.zs

* Extra-Utilities.zs

* Forbidden-Magic.zs

* Forestry.zs

* GT++.zs

* Galacticraft.zs

* Genetics.zs

* Gravisuite.zs

* Gregtech.zs

* HardcoreEnderExpansion.zs

* Harvestcraft.zs

* Create OreDictHelper.java

* IC2.zs

* Magic-Bees.zs

* Minecraft.zs

* Natura.zs

* Project_Red.zs

* Railcraft.zs

* Renaming.zs

* Steves-Carts.zs

* Thaumcraft-01-Basic.zs Thaumcraft-03-Alchemy.zs Thaumcraft-05-Golemancy.zs

* Thaumic-Bases-01-Main.zs

* ThaumicTinkerer-01.zs

* Tinkers-Construct.zs

* Tinkers-Defence.zs

* ZZ-Client-Only.zs

* Create AvaritiaHelper.java

* Blood Magic helper

* Update BloodMagicHelper.java

* AE2-Wireless-Crafting.zs

* AE2.zs

* Avaritia.zs

* Blood-Arsenal.zs

* Blood-Magic.zs

* Compact-Kinetic_generators.zs

* Draconic-Evolution.zs

* Ender-IO.zs

* Forbidden-Magic.zs

* Galacticraft.zs

* Gendustry.zs

* Gregtech.zs

* IC2.zs

* Open-Computers.zs

* Project_Red.zs

* SGCraft.zs

* Steves-Carts.zs

* Missed blood orb recipes

* Add NBT to custom tooltips

* Deps1

* Remove wildcard imports from scripts

* Fix recipes from https://github.com/GTNewHorizons/NewHorizonsCoreMod/pull/260

* Use mods enum instead of strings

* Fix error: Code too large

* Fix error: Method too large

* ZZ-Client-Only

* Merge

* Fix dep

* Fix QED recipes

* Fix rolling recipes

* Remove TemporaryMods.java

* Add https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/commit/c4ca2b68e0baac9ead22a164b45a74ffa1454e49

* Fix ZZ-Client-Only

* Add https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/commit/fbb1c6514ddd5793ab299825e7f6435834241e4f

kuba6000 committed:
Remove recipes (#578)

kuba6000 committed:
Add missing centrifuge and extractor recipes (#579)

chochem committed:
integrate replace for LuV machines (#580)

Co-authored-by: Martin Robertz <dream-master@gmx.net>

Dream-Master committed:
update bs+dep

Dream-Master committed:
remove ec2 + BQ warning
